build.gradle

```diff
@@ -522,7 +574,9 @@ dependencies {
     runtimeOnly 'com.eclipsesource.minimal-json:minimal-json:0.9.5'
     runtimeOnly 'commons-codec:commons-codec:1.16.0'
     runtimeOnly 'org.cryptacular:cryptacular:1.2.6'
-    runtimeOnly 'com.google.errorprone:error_prone_annotations:2.21.1'
+
+    compileOnly 'com.google.errorprone:error_prone_annotations:2.21.1'
```

Comment:
it has an annotation process which triggers during compilation

Reply:
I think this might have been marked runtime only to make sure the output of the security plugin dependencies was identical when the gradle upgrade was performed (pretty sure I made this change).  Both of the runtimeOnly -> compileOnly changes seem safe not to include in the output jar. 👍

Reply:
yes and if you use `runtimeOnly` it does not trigger annotation processors as result compilation failed. Since both libraries use maven the scope for both are `provided` which is `compileOnly` in case of gradle.